samples/lorawan/class_a/src/main.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Class A LoRaWAN sample application
+ *
+ * Copyright (c) 2019 Manivannan Sadhasivam <mani@kernel.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <lorawan.h>
+#include <zephyr.h>
+
+/* Customize based on network configuration */
+#define LORAWAN_DEV_EUI			{ 0xDD, 0xEE, 0xAA, 0xDD, 0xBB, 0xEE,\
+					  0xEE, 0xFF }
+#define LORAWAN_JOIN_EUI		{ 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,\
+					  0x00, 0x00 }
+#define LORAWAN_APP_EUI			{ 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,\
+					  0x00, 0x00 }
+#define LORAWAN_APP_KEY			{ 0x2B, 0x7E, 0x15, 0x16, 0x28, 0xAE,\
+					  0xD2, 0xA6, 0xAB, 0xF7, 0x15, 0x88,\
+					  0x09, 0xCF, 0x4F, 0x3C }
+#define LORAWAN_DEFAULT_DATARATE	LORAWAN_DR_0
+
+#define MAX_DATA_LEN 10
+#define DELAY 5000
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(lorawan_class_a);
+
+char data[MAX_DATA_LEN] = {'h', 'e', 'l', 'l', 'o', 'w', 'o', 'r', 'l', 'd'};
```

Comment:
Could this be done other way around, so just set the `data` variable (without size) and if you need its length, the use `sizeof()`, like this:
```
char data[] = {'h', 'e', 'l', 'l', 'o', 'w', 'o', 'r', 'l', 'd'};
#define MAX_DATA_LEN sizeof(data)
```

Reply:
Okay